### ISSUE TITLE
On token asset page, set some metadata for sharing

### ISSUE THREAD

**Issue by treeder** [gochain-io/explorer#254]
This may be harder than it sounds since I think it would have to delivered server side (I think angular has some features to server side render which we could look at?). 

Updating metadata, but I think updating at this point may be too late (hence server side): 
https://stackoverflow.com/questions/41454978/dynamic-ogimage-angular2

Set og:image for facebook: https://stackoverflow.com/a/46139611/105562

Set up a nice twitter card: 
https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#opengraph

**Comment by bergauz** [comment]
@treeder yes, it must be SSR, https://stackoverflow.com/questions/45262719/angular-4-update-meta-tags-dynamically-for-facebook-open-graph, so we need to add nodejs as renderer

**Comment by bergauz** [comment]
btw what should be in token page?
